docs/content/documentation/content/image-processing/index.md

```diff
@@ -118,7 +121,7 @@ The `resize_image()` can be used multiple times and/or in loops. It is designed
 
 This can be used along with `assets` [page metadata](@/documentation/templates/pages-sections.md) to create picture galleries.
 The `assets` variable holds paths to all assets in the directory of a page with resources
-(see [assets colocation](@/documentation/content/overview.md#assets-colocation)): if you have files other than images you
+(see [asset co-location](@/documentation/content/overview.md#asset-co-location)); if you have files other than images you
```

Comment:
I think both spelling are correct (at least according to Wikipedia: https://en.wikipedia.org/wiki/Colocation_centre) and not changing it means that links to it will still work.

Reply:
Yes, you are right. There is also "collocation" (older word). Please see https://www.grammarphobia.com/blog/2013/08/collocation-colocation-co-location.html long discussion. I just thought "co-location" is a bit easier to read (I think I fixed the links too). I'm ok with colocation, so please revert this change.